src/util/authenticationGuard.js

```diff
@@ -5,14 +5,16 @@ import authenticationQuery from '@/graphql/query/authenticationQuery.graphql';
 const isServer = typeof window === 'undefined';
 
 // Return true if user logged in recently enough and false otherwise
-const checkLastLoginTime = (data, durationKey, defaultDuration) => {
-	const lastLogin = _get(data, 'my.lastLoginTimestamp', 0);
-	const duration = 1000 * (parseInt(_get(data, `general.${durationKey}.value`), 10) || defaultDuration);
+export function checkLastLoginTime(data, durationKey, defaultDuration) {
```

Comment:
Export this function so we can import it into monthly good setup page and also refactor `_get` from it.